test/system-test/utils.py

```diff
@@ -173,10 +177,42 @@ def vote(proposal_id, vote, get_logs=False):
 
-
-def get_test_wrapping_key():
-    return '"-----BEGIN PUBLIC KEY-----\\nMIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEA0L9FDBjydkdstv7OKqkw\\ndMiugRqlSHC9Lchfd7jh5uCzv602LhlBJQeEFYchvaEquISLQFoZxEkpGEbEb15v\\nN2dKwTCi0ioEGBidtFmuKiVZqf46Hbnw4OdinQHrlGO2PRsRE+DYPOy6xrZTKEnD\\nK+OnHDsZ0U2qNJ80IjbxcC83lQpaMx8Ij8AddGY9Msv0TMMgaVsrDaQYLC8tmJih\\nxVI2f5BHFbTy3pw4Sq9xp6se+S/ycOUF0M6RFArZcD9uR/NxJKFJJlGOKskgwLnT\\nlhPck4sGeLhHUydRdqw0+h8EDsTIUysNxMOYtZETaVuBS+ISMy8+WQgVPs12Ujr3\\n17kaHeZr8Lq0bwaFHruRBpNwqtUCBv57IBpe7hnEDDdvOvN/tPubf1dv3HxEt42T\\nqqfozS+a9+1hcI8hpNlEjh+qcy1BmhSOXvmRzlhauX4xv6OLCNkRxo6x2Q/1moDC\\nPgnaJJVIuESr07xnC8fk43i5qFzEXQO3hwNsjd7sqFBBTb6t5N6Nm37mMNTqsDky\\nniAeFG+1gK/UD+cMPfbUIDaCqpCDwrTX0gMqqUTDG6eNPmQaOa+slici3h9WLaNy\\nmEzfKqJMBggKib/+e4Eb/ENdvxeT1X2YXpZ3tjZE+bRoiDgN4FYqBzYtZ/ieRcsq\\n4fPqgZPbh+ivT2o7QutzWH0CAwEAAQ==\\n-----END PUBLIC KEY-----\\n"'  # pragma: allowlist secret
-
+def get_test_public_wrapping_key():
+    with open(os.path.join(os.path.dirname(__file__), '../data-samples/publicWrapKey.pem'), 'r') as file:
+        key = file.read()
+        # Escape newlines and wrap the key in double quotes
+        key = key.replace("\n", "\\n")
+        #return f"{key}"
+        return f'"{key}"'
+
+def get_test_private_wrapping_key():
+    with open(os.path.join(os.path.dirname(__file__), '../data-samples/privateWrapKey.pem'), 'r') as file:
```

Comment:
Rather than two separate files for private and public key, could we have a single private key file and then for the get public key function we can derive the public key from the private key?

This would prevent the possibility of accidentally desyncing the two

Reply:
Is possible but not so obvious for PEM. With JWT it is very easy to do. 
We also use these files in bash scripts which would overcomplicate these.

Reply:
In bash:
```
openssl rsa -in test/data-samples/privateWrapKey.pem -pubout
```

In python
```
RSA.import_key(get_test_private_wrapping_key()).publickey().export_key("PEM").decode()
```

Reply:
it is a nice to have. We have several safety measures to detect a mismatch between private and public key. These new added tests will fail and also tests in privacy-sandbox-dev will fail.
My current priority is to unblock Muhamed. Let's move forward with the current implementation.

Reply:
Sure